colors/monokai.vim

```diff
@@ -1 +0,0 @@
-lua require('monokai')
```

Comment:
We should change this to `lua require('monokai').setup()`, this will allow the plugins to be used regardless of whether or not users call the `setup` method.

Users who want to use a different flavour of monokai can call the following in their dotfiles:
```
local monokai = require('monokai')
monokai.setup(monokai.palette_pro)
```

Reply:
Fixed in b401f9c